stylus/include/Footer.styl

```diff
@@ -0,0 +1,30 @@
+footer {
```

Comment:
No need to use brackets in `.styl` files.  Also, try to avoid `px` styling in favor of `rem` units.

Reply:
Also, use `.Footer` in this file instead of `footer`